### PR TITLE
[Queues] Add Metrics page to Queues, GraphQL Examples

### PR DESCRIPTION
**Pull request by mtlemilio**

### CODE REVIEW

**Review by elithrar — APPROVED** [review]
Approved conditional on CI passing.

File: content/queues/learning/metrics.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+pcx_content_type: concept
+title: Metrics 
+weight: 5
+---
+
+# Metrics
+
+You can view the metrics for a Queue on your account via the dashboard. Navigate to **Workers** > **Queues** > **your Queue** and under the **Metrics** tab you'll be able to view line charts describing the number of messages processed by final outcome, the number of messages in the backlog, and other important indicators.
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/metrics.md]:
```suggestion
You can view the metrics for a Queue on your account via the Cloudflare dashboard. Navigate to **Workers** > **Queues** > **your Queue** and under the **Metrics** tab you'll be able to view line charts describing the number of messages processed by final outcome, the number of messages in the backlog, and other important indicators.
```

File: content/queues/learning/metrics.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+The metrics displayed in the dashboard charts are all pulled from Cloudflare's GraphQL Analytics API, and you can access them programmatically.
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/metrics.md]:
```suggestion
The metrics displayed in the Cloudflare dashboard charts are all pulled from Cloudflare's GraphQL Analytics API. You can access the metrics programmatically.
```

File: content/queues/learning/metrics.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+## Writing GraphQL Queries
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/metrics.md]:
```suggestion
## Write GraphQL queries
```

File: content/queues/learning/metrics.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+Here are some GraphQL examples to get you started exploring your Queues metrics.
+
+Learn more about the GraphQL Analytics API [here](/analytics/graphql-api/).
+
+The Queues metrics are split across three different nodes under `viewer` > `accounts`. To learn more about how to navigate the GraphQL schema and discover which data are available and you can read [this](/analytics/graphql-api/getting-started/explore-graphql-schema/).
+
+### Getting Average Queue Backlog over Time Period
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/metrics.md]:
```suggestion
### Get average Queue backlog over time period
```

File: content/queues/learning/metrics.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+          messages
+          bytes
+        }
+      }
+    }
+  }
+}
+```
+
+### Getting Average Consumer Concurrency by Hour
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/metrics.md]:
```suggestion
### Get average consumer concurrency by hour
```

File: content/queues/learning/metrics.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+        dimensions {
+          datetimeHour
+        }
+      }
+    }
+  }
+}
+```
+
+### Getting Message Operations by Minute
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/metrics.md]:
```suggestion
### Get message operations by minute
```

File: content/queues/learning/metrics.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+
+# Metrics
+
+You can view the metrics for a Queue on your account via the dashboard. Navigate to **Workers** > **Queues** > **your Queue** and under the **Metrics** tab you'll be able to view line charts describing the number of messages processed by final outcome, the number of messages in the backlog, and other important indicators.
+
+The metrics displayed in the dashboard charts are all pulled from Cloudflare's GraphQL Analytics API, and you can access them programmatically.
+
+## Writing GraphQL Queries
+
+Here are some GraphQL examples to get you started exploring your Queues metrics.
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/metrics.md]:
```suggestion
Examples of how to explore your Queues metrics.
```

File: content/queues/learning/metrics.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+title: Metrics 
+weight: 5
+---
+
+# Metrics
+
+You can view the metrics for a Queue on your account via the dashboard. Navigate to **Workers** > **Queues** > **your Queue** and under the **Metrics** tab you'll be able to view line charts describing the number of messages processed by final outcome, the number of messages in the backlog, and other important indicators.
+
+The metrics displayed in the dashboard charts are all pulled from Cloudflare's GraphQL Analytics API, and you can access them programmatically.
+
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/metrics.md:14]:
```suggestion
The Queues metrics are split across three different nodes under `viewer` > `accounts`. Refer to [Explore the GraphQL schema](/analytics/graphql-api/getting-started/explore-graphql-schema/) to learn how to navigate a GraphQL schema and discover which data are available.

To learn more about the GraphQL Analytics API, refer to [GraphQL Analytics API](/analytics/graphql-api/).

```

File: content/queues/learning/metrics.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+## Writing GraphQL Queries
+
+Here are some GraphQL examples to get you started exploring your Queues metrics.
+
+Learn more about the GraphQL Analytics API [here](/analytics/graphql-api/).
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/metrics.md]:
```suggestion
```

Comment (Maddy-Cloudflare) [reply content/queues/learning/metrics.md]: I moved 

**Learn more about the GraphQL Analytics API [here](/analytics/graphql-api/).**

above.

File: content/queues/learning/metrics.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+
+The Queues metrics are split across three different nodes under `viewer` > `accounts`. To learn more about how to navigate the GraphQL schema and discover which data are available and you can read [this](/analytics/graphql-api/getting-started/explore-graphql-schema/).
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/metrics.md]:
```suggestion
```

Comment (Maddy-Cloudflare) [reply content/queues/learning/metrics.md]: Also moved this above.

**Review by Maddy-Cloudflare — APPROVED** [review]

File: content/queues/learning/metrics.md [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+The Queues metrics are split across three different nodes under `viewer` > `accounts`. Refer to [Explore the GraphQL schema](/analytics/graphql-api/getting-started/explore-graphql-schema/) to learn how to navigate a GraphQL schema and discover which data are available.
+
+To learn more about the GraphQL Analytics API, refer to [GraphQL Analytics API](/analytics/graphql-api/).
+
+## Write GraphQL queries
+
+Examples of how to explore your Queues metrics.
+
+
+
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/metrics.md]:
```suggestion
```

File: content/queues/learning/metrics.md [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+The metrics displayed in the Cloudflare dashboard charts are all pulled from Cloudflare's GraphQL Analytics API. You can access the metrics programmatically.
+The Queues metrics are split across three different nodes under `viewer` > `accounts`. Refer to [Explore the GraphQL schema](/analytics/graphql-api/getting-started/explore-graphql-schema/) to learn how to navigate a GraphQL schema and discover which data are available.
+
+To learn more about the GraphQL Analytics API, refer to [GraphQL Analytics API](/analytics/graphql-api/).
+
+## Write GraphQL queries
+
+Examples of how to explore your Queues metrics.
+
+
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/metrics.md]:
```suggestion
```

Comment (Maddy-Cloudflare) [reply content/queues/learning/metrics.md]: Remove these two lines.

File: content/queues/learning/metrics.md [hunk 13]
```diff
@@ -0,0 +1,99 @@
+---
+pcx_content_type: concept
+title: Metrics 
+weight: 5
+---
+
+# Metrics
+
+You can view the metrics for a Queue on your account via the Cloudflare dashboard. Navigate to **Workers** > **Queues** > **your Queue** and under the **Metrics** tab you'll be able to view line charts describing the number of messages processed by final outcome, the number of messages in the backlog, and other important indicators.
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/metrics.md]:
```suggestion
You can view the metrics for a Queue on your account via the [Cloudflare dashboard](https://dash.cloudflare.com). Navigate to **Workers** > **Queues** > **your Queue** and under the **Metrics** tab you'll be able to view line charts describing the number of messages processed by final outcome, the number of messages in the backlog, and other important indicators.
```

File: content/queues/learning/metrics.md [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+
+The metrics displayed in the Cloudflare dashboard charts are all pulled from Cloudflare's GraphQL Analytics API. You can access the metrics programmatically.
```
Comment (Maddy-Cloudflare) [inline content/queues/learning/metrics.md:11]:
```suggestion
The metrics displayed in the Cloudflare dashboard charts are all pulled from Cloudflare's GraphQL Analytics API. You can access the metrics programmatically.

```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Pull request merged**